logic.py

```
@@ -762,7 +762,83 @@ def _is_candidate_valid(col):
-    return html_table, or_results, aor_results, interaction_results
+    # --- START FIX: Forest Plot Generation ---
+    forest_plot_html = ""
+    try:
+        # Determine which results to plot (Priority: AOR > OR)
+        plot_data = []
+        source_results = aor_results if aor_results else or_results
+        title_prefix = "Adjusted" if aor_results else "Crude"
+        
+        for label, metrics in source_results.items():
+            # Skip rows without proper data
+            if not isinstance(metrics, dict): continue
```

Ruff (0.14.10)

Multiple statements on one line (colon)

(E701)